doc/conf.py

```diff
@@ -101,6 +101,8 @@ def __getattr__(cls, name):
 breathe_projects = {"Lightning-Qubit": "./doxyoutput/xml"}
 breathe_default_project = "Lightning-Qubit"
 
+mathjax_path = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
```

Comment:
Do wee need this to be specified?

Reply:
Yep. The identity gate symbol does not correctly display without this.